recipes/hover/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
It would be nice to create an upstream issue to add the license to the distribution. (Maybe there is already one, then never mind.)

Reply:
The upstream did indicate MIT License, but there was no LICENSE.txt in the tarball.

Reply:
Yes, it's a common problem in sdist tarballs.